### PR TITLE
docs(backing_up_data): Add community tools for reference

### PR DESCRIPTION
**Pull request by croemmich**
The page with contributed tools is a bit hard to find. Maybe include the backup/restore tools on the backup page?

### CODE REVIEW

**Comment by carmstrong** [comment]
The potential issue here is the burden of responsibility: if something is in our documentation proper, we ensure that it works well and is updated to reflect changes in Deis. If it's a third-party tool, putting it in `contrib/` implies that it is a community tool that is unsupported by us.

Perhaps the middle ground is to put it in `contrib/` which we are doing in #3905, and then including something in the official backup documentation like "Various community members have developed tools to backup and restore Deis" with a link to the contrib README?

**Comment by croemmich** [comment]
@carmstrong I think that's a reasonable solution. Thanks!

**Comment by croemmich** [comment]
@carmstrong, I've updated the docs, let me know what you think.

File: docs/managing_deis/backing_up_data.rst [hunk 1]
```diff
@@ -217,6 +217,13 @@ use in the ``export`` command should correspond to the IP of the host machine wh
 
 That's it! The cluster should be fully restored.
 
+Tools
+-----
+
+Various community members have developed tools to assist in automating the backup and restore process outlined above,
```
Comment (carmstrong) [inline docs/managing_deis/backing_up_data.rst]: I would split this into a separate sentence, but otherwise this is exactly what I was thinking. :+1:

**Comment by croemmich** [comment]
@carmstrong, updated per your suggestion. Thanks!

**Comment by carmstrong** [comment]
LGTM. This passed the relevant docs tests (and saw the userland proxy error which is unrelated): https://ci.deis.io/job/test-ec2/1477/consoleFull#682192135502066c60b2236c09cc9bf8

**Comment by mboersma** [comment]
LGTM!

**Pull request merged**